src/components/preferences/PreferencesPage.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import styles from "./styles";
+import PreferencesTab from "./PreferencesTab";
+import ShortcutsTab from "./ShortcutsTab";
+import { DialogActions, Button } from "@material-ui/core";
+import { Formik, Form } from "formik";
+import { DEConfirmationDialog, LoadingMask } from "@cyverse-de/ui-lib";
+
+const useStyles = makeStyles(styles);
+
+export default function PreferencesPage(props) {
+    const [restoreDef, setRestoreDef] = useState(false);
+    const classes = useStyles();
+    const { config } = props;
+
+    const handleSubmit = (values, actions) => {
+        actions.setSubmitting = true;
+        console.log(values);
+    };
+
+    const restoreDefaults = (setFieldValue) => (event) => {
+        setFieldValue("preferences.rememberLastPath", true);
+        setFieldValue("preferences.saveSession", true);
+        setFieldValue("preferences.enableImportEmailNotification", true);
+        setFieldValue("preferences.enableWaitTimeMessage", true);
+        setFieldValue("preferences.enableAnalysisEmailNotification", true);
+        setFieldValue("preferences.enableHPCPrompt", true);
+        setFieldValue("preferences.notificationKBShortcut", "N");
+        setFieldValue("preferences.dataKBShortcut", "D");
+        setFieldValue("preferences.closeKBShortcut", "Q");
+        setFieldValue("preferences.appsKBShortcut", "A");
+        setFieldValue("preferences.analysisKBShortcut", "Y");
+        // TODO find a way to get users default output folder
+        //setFieldValue("preferences.default_output_folder", )
+        setRestoreDef(false);
+    };
+
+    return (
+        <div className={classes.root}>
+            <Formik initialValues={config} onSubmit={handleSubmit}>
```

Comment:
This is not necessary, just something to note - since we're not putting webhooks on this page anymore, you could actually set initialValues to `config.preferences` since everything you need is in the `preferences` object.  And then for the rest of your component, everywhere you have `name="preferences.*"`, you could remove `preferences.`.  Again, not necessary, what you've got completely works fine, just thought I'd mention it.